compiler/env/OMRCPU.cpp

```diff
@@ -156,6 +156,18 @@ OMR::CPU::detect(OMRPortLibrary * const omrPortLib)
    return TR::CPU(processorDescription);
    }
 
+TR::CPU
+OMR::CPU::detectRelocatable(OMRPortLibrary * const omrPortLib)
```

Comment:
`detectRelocatable` is the same as `detect`. Will it change in the future? What is actually supposed to do?

Reply:
we have a corresponding `J9::CPU::detectRelocatable` in OpenJ9 that changes the behavior. I'm not sure if the detectRelocatable logic should reside in OMR or OpenJ9 (currently it makes more sense to reside in OpenJ9 during the development phase and easier for us to change processors). The only purpose OMR::detectRelocatable has currently is to act as the base api for J9::detectRelocatable.